<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFrame</class>
 <widget class="QDialog" name="MainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>240</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../sb3000.qrc">:/images/res/bg-main.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblBackgroundFrame">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>10</y>
     <width>297</width>
     <height>217</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../sb3000.qrc">:/images/res/bg-menu.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>6</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../sb3000.qrc">:/images/res/title-1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>8</y>
     <width>273</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleIcon">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>10</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatusIcon">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>215</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../sb3000.qrc">:/images/res/icon-help.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lblStatusText">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>275</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Arrow &amp; OK, ESC for return</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitleText1">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>10</y>
     <width>110</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../sb3000.qrc"/>
 </resources>
 <connections/>
</ui>
